packages/rum/src/domain/rumEventsCollection/resource/matchRequestTiming.ts

```diff
@@ -24,8 +24,13 @@ export function matchRequestTiming(request: RequestCompleteEvent) {
   if (!performance || !('getEntriesByName' in performance)) {
     return
   }
-  const candidates = performance
-    .getEntriesByName(request.url, 'resource')
+  const sameNameEntries = performance.getEntriesByName(request.url, 'resource')
+
+  if (!(sameNameEntries.length > 0 && 'toJSON' in sameNameEntries[0])) {
```

Comment:
NIT: I'd find this clearer:
```suggestion
  if (!sameNameEntries.length || !('toJSON' in sameNameEntries[0])) {
```